pkg/jaeger/jaeger_test.go

```diff
@@ -0,0 +1,109 @@
+package jaeger
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/emicklei/go-restful/v3"
+	"github.com/opentracing/opentracing-go"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	jaegerclientgo "github.com/uber/jaeger-client-go"
+)
+
+func TestJaegerFilterWithZipkinMultipleHeaders(t *testing.T) {
```

Comment:
I guess this func name is random am I correct you can go for a more reliable name!

Reply:
Hey @Codecollab-go, that's not a random name `TestJaegerFilterWithZipkinMultipleHeaders` is for the test functioning of the Jaeger.